src/leveled_sst.erl

```diff
@@ -78,6 +78,10 @@
 -define(NOLOOK_SLOTSIZE, 256).
 -define(NOLOOK_BLOCKSIZE, {56, 32}). % 4x + y = ?NOLOOK_SLOTSIZE
 -define(COMPRESSION_LEVEL, 1).
+    % When using native compression - how hard should the compression code
+    % try to reduce the size of the compressed output. 1 Is to imply minimal
+    % effort, 6 is default:
```

Comment:
but not the default of leleled, because here 1 is default

Reply:
Changed to COMPRESSION_FACTOR as this is different to the configurable COMPRESSION_LEVEL.  Will add reference to default being OTP not leveled.